### ISSUE TITLE
Layout crush when using component as swipe item

### ISSUE THREAD

**Issue by littlee** [voronianski/react-swipe#100]
## Description
When using component as swipe item, the layout will be crushed.

## Demo link
I don't know how to save the boilerplate to another link, so I paste the code here

```js
class Item extends React.Component {
  render() {
    return (
      <div className="item">{this.props.num}</div>
      )
  }
}

class Carousel extends React.Component {
  constructor() {
    super();
    
    this.state = {
        slides: ['pane 1', 'pane 2', 'pane 3']
    };
  }    
  
  componentDidUpdate(prevProps, prevState) {
    // due to buggy iframe behavior
    window.dispatchEvent(new Event('resize'));
  }
  
  next() {
    this.refs.reactSwipe.next();
  }

  prev() {
    this.refs.reactSwipe.prev();
  }
  
  addSlide() {
    const slides = this.state.slides;
    slides.push(`pane ${slides.length + 1}`);
    this.setState({ slides });
  }

  render() {
    return (
      <div>
        <ReactSwipe ref="reactSwipe" className="carousel">
          {this.state.slides.map((slide) => {
            return (
              <Item key={slide} num={slide}/>
            );
          })}
        </ReactSwipe>

        <div>
          <button type="button" onClick={::this.prev}>Prev</button>
          <button type="button" onClick={::this.next}>Next</button>
          <button type="button" onClick={::this.addSlide}>Add Slide</button>
        </div>
      </div>
    );
  }
}

ReactDOM.render(
  <Carousel />,
  document.getElementById('app')
);
```
![qq20161222-0](https://cloud.githubusercontent.com/assets/7534668/21413550/173e5634-c833-11e6-9ec3-b7f39c20b2f6.png)

**Comment by dohyeondk** [comment]
I found the same.

**Comment by littlee** [comment]
And I find that the solution is wrap the `<Item/>` with `<div>` tag
```js
<ReactSwipe ref="reactSwipe" className="carousel">
  {this.state.slides.map((slide) => {
    return (
      <div>
        <Item key={slide} num={slide}/>
      </div>
    );
  })}
</ReactSwipe>
```

**Comment by evilguc** [comment]
@littlee it is likely you had [this isue](https://github.com/voronianski/react-swipe/pull/102#issue-210810664). And yes, using primitive wrapper like `<div>` can be used as a workaround until fixed in code base.

**Comment by voronianski** [comment]
@evilguc @littlee @kimshako should be fixed, new version is `5.0.7`.
